Single slide indicator condition issue504

Debanshu777 opened the pull request:
#504 > Do not show position dot if there is only one slide

I resolved it with the issue by adding two conditions in
..\AppIntro\appintro\src\main\java\com\github\appintro\indicator\DotIndicatorController.kt

cortinico commented on appintro/src/main/res/values-bn-rIN/strings.xml:
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_intro_skip_button">এড়িয়ে যান</string>
```
This file should be outside of your diff

Debanshu777 commented:
I have added the changes as suggested @cortinico 
> This file should be outside of your diff
Reverted the last two commits 8f695df & 1a07479 and adding a new modified commit b327651

cortinico commented on appintro/src/main/java/com/github/appintro/indicator/DotIndicatorController.kt:
```diff
@@ -46,6 +46,7 @@ class DotIndicatorController(context: Context) : IndicatorController, LinearLayo
         this.slideCount = slideCount
 
         for (i in 0 until slideCount) {
+            if (slideCount == 1) { break }
```
I would suggest you instead change this to (you need to move it down):

```suggestion
            if (slideCount == 1) { 
                dot.visibility = View.INVISIBLE
            }
```

This will make sure that the dot is still added to the hierarchy and it's hidden.
You can then amend the change to lines 71-73

Debanshu777 replied:
Okay I got the point, I will make the required changes.

cortinico commented on appintro/src/main/java/com/github/appintro/indicator/DotIndicatorController.kt:
```diff
@@ -67,7 +70,12 @@ class DotIndicatorController(context: Context) : IndicatorController, LinearLayo
                 unselectedIndicatorColor
             }
-            (getChildAt(i) as ImageView).let { DrawableCompat.setTint(it.drawable, tint) }
+            (getChildAt(i) as ImageView).let {
+                DrawableCompat.setTint(it.drawable, tint)
+            }
+            if (slideCount == 1) {
+                (getChildAt(i) as ImageView).visibility = View.INVISIBLE
+            }
```
```suggestion
            (getChildAt(i) as ImageView).let { DrawableCompat.setTint(it.drawable, tint) }
```

Those changes are unnecessary and can probably be removed.

cortinico reviewed (approved):
Thanks for the contribution @Debanshu777 🙏

cortinico commented:
Fixes #504
